features/job_seeker.feature

```
@@ -20,6 +27,7 @@ Scenario: new Js Registration
   And I fill in "Password" with "password"
   And I fill in "Password Confirmation" with "password"
   And I fill in "Year Of Birth" with "1990"
+  Then I select "Unemployedlooking" in select list
```

I do not thinking defaulting one of the values would be good idea, this can be misleading to employer or jobseeker. I would just prompt the jobseeker maybe asking to select the status options and without selecting would show error while submitting the form.